CheckResult.java

```diff
@@ -1,7 +1,7 @@
 package psyh;
```

Comment:
change the name of the package

Reply:
I won't rename "psyh". Let's close this question.